src/fiatconnect/ReviewScreen.test.tsx

```diff
@@ -0,0 +1,142 @@
+import {
+  CryptoType,
+  FiatAccountSchema,
+  FiatAccountType,
+  FiatType,
+} from '@fiatconnect/fiatconnect-types'
+import { render } from '@testing-library/react-native'
+import * as React from 'react'
+import { Provider } from 'react-redux'
+import FiatConnectReviewScreen from 'src/fiatconnect/ReviewScreen'
+import { CICOFlow } from 'src/fiatExchanges/utils'
+import { Screens } from 'src/navigator/Screens'
+import { createMockStore, getMockStackScreenProps } from 'test/utils'
+
+function getProps(flow: CICOFlow, fee?: string, image?: string) {
+  return getMockStackScreenProps(Screens.FiatConnectReview, {
+    flow,
+    cicoQuote: {
```

Comment:
There's a `getQuotes` helper for creating CicoQuote mock data used in PaymentMethodSelection.test.tsx that maybe can be reused here?

Reply:
I don't think that helper is compatible with FiatConnect quote response, which is what this component uses. @jh2oman we wanted to directly use the fiat connect quote response type here as a prop right?